lib/src/impl_interface/impl_interface.dart

```diff
@@ -17,9 +17,12 @@ library impl_stub;
 import 'dart:typed_data';
 import 'dart:async';
 
+import '../webcrypto/webcrypto.dart';
```

Comment:
What we don't want this. That creates cyclic imports. It becomes messy, let's avoid that.

We can just make a `KeyPairImpl`, or fully move `KeyPair` into `impl_interface.dart`.

Reply:
Yes, this makes sense. tysm. modifying accordingly

Reply:
Or we can just do as proposed in:
https://github.com/google/webcrypto.dart/pull/145/files#r1804350232

Specifcally, we do:
```
Future<(EcdhPrivateKeyImpl, EcdhPublicKeyImpl)> generateKey(EllipticCurve curve);
```

Then we can still make it into a `KeyPair` in `webcrypto/webcrypto.dart` where we have to wrap `EcdhPrivateKeyImpl` into `EcdhPrivateKey`.